src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Todo[];
+  onSelectTodo: (todo: Todo) => void;
+  onClosedTodo: Todo | null;
```

Comment:
Change the type of `onClosedTodo` to `(todo: Todo | null) => void;`. This is because it should be a callback function with a nullable `Todo` argument.

Reply:
but selectedTodo, which i share in App - is Todo | null, not a function